### DISCUSSION TITLE
How to burn bootloader at LGT ?

### DISCUSSION THREAD

**Discussion by herbkonik** [dbuezas/lgt8fx#315] [Q&A] [answered]
Hi all,
it seams i have demaged the bootloader at 2 of my boards, because it shows a unusual behavior and output if i upload a sketch:
The upload itself works, bord makes what is expected, but after upload the IDE shows a error message "can't verify upload, programmer is not responding".
Do you know where i can find the file for the bootloader?
And: Is it the same procedere as at the Arduino boards ? (I think so, just to make sure...)

**Comment by dbuezas** [comment, marked as answer]
Here's the tutorial: https://github.com/dbuezas/lgt8fx/blob/master/lgt8f/libraries/LarduinoISP/readme.md

**Comment by herbkonik** [comment]
Thank you very much

**Comment by 11armatur** [comment]
How make LGTSWD?

**Reply by hmeijdam** [reply]
https://github.com/LGTMCU/LarduinoISP

**Comment by dbuezas** [comment]
See here: 
https://github.com/dbuezas/lgt8fx/blob/master/lgt8f/libraries/LarduinoISP/readme.md

The source for the isp is already built-in in this Arduino core, you can access in `File/Examples/LarduinoISP/LarduinoISP` inside the Arduino IDE

**Comment by i2bel** [comment]
https://github.com/i2bel/esp32c3-2-LGT8F328P-bootloader-writer
